hoomd/hpmc/compute.py

```diff
@@ -295,28 +348,80 @@ def _attach_hook(self):
     @log(category='sequence', requires_run=True)
     def sdf(self):
         """(*N_bins*,) `numpy.ndarray` of `float`): :math:`s[k]` - The scale \
-        distribution function :math:`[\\mathrm{probability\\ density}]`.
+        distribution function for compression moves \
+        :math:`[\\mathrm{probability\\ density}]`.
+
+        .. deprecated:: v3.8.0
+            Use `sdf_compression`.
+        """
+        self._cpp_obj.compute(self._simulation.timestep)
+        return self._cpp_obj.sdf_compression
```

Comment:
```suggestion
        return self.sdf_compression
```
Reuse existing code.